docs/devel/translations.rst

```diff
@@ -17,10 +17,42 @@ This can be done using :guilabel:`Start new translation` in :ref:`component`.
 
 .. note::
 
-    Project admins can always start translation within Weblate directly.
```

Comment:
For adding a translation the note was actually appropriate because it was opposed to that the fact that a user can't always add a translation directly.

Reply:
That's a valid point - admins can add translation when it's technically possible to do so, while users only when component is configured to allow this.